src/mobx/model/network/eth.network.ts

```diff
@@ -395,6 +397,26 @@ const ethVaultDefinitions: BadgerVault[] = [
       decimals: 18,
     },
   },
+  {
+    depositToken: {
```

Comment:
@mrbasado lets make a ticket to deprecate this functionally and utilize either pure API (no connected provider) or registry verification for definitions - (post filter on api) or fallback. 

This will allow us to have no definitions in the UI for future